### PR TITLE
Updated Mid Term Eval Blog for GSOC 2022 Final Submission

### PR DESCRIPTION
**Pull request by guneetsk99**
@DalilaSalamani 
Here is the updated blog for the HSF page with all the screen shots.
Let me know if any changes would be required

### CODE REVIEW

**Review by agheata — CHANGES_REQUESTED** [review]
@guneetsk99 please remove the duplicated `Geant4-FastSim` from the title. To make the photo visible, please add the line: `photo: blog_authors/GuneetSingh.jpg` in the header part. You have quite exhaustive text, quite difficult to follow, but a very short introduction. I suggest replacing the intro with a longer sentence describing your work. @DalilaSalamani we will need your approval before merging.

**Comment by guneetsk99** [comment]
@agheata Made the suggested changes

**Comment by guneetsk99** [comment]
> @guneetsk99 please remove the duplicated `Geant4-FastSim` from the title. To make the photo visible, please add the line: `photo: blog_authors/GuneetSingh.jpg` in the header part. You have quite exhaustive text, quite difficult to follow, but a very short introduction. I suggest replacing the intro with a longer sentence describing your work. @DalilaSalamani we will need your approval before merging.

On it @agheata

**Comment by DalilaSalamani** [comment]
@guneetsk99 I think you need to do more polishing and restructuring. 

General comments 
- Please only keep few titles to have a nice structure of the blog so it can be easy to understand
- Please remove all the very specific details (these we can put them in our website). Here we want to have a summarised description of the project which is easy to follow

Some more specific comments
# Introduction
- Geant -> part of the Geant4 fast simulation group
- Kubeflow Pipeline for training ML Fast Simulation -> Kubeflow Pipeline for training machine learning based model for fast shower simulation 
- I think the section from "This report is an indepth documentation .... to  It also provides some guidance in handling docker images though the user is expected to have some basic familiarity with the docker concepts" can be removed 
# Outcomes from GSOC 2022
- Here the order can be re-done, for instance you can put the pipeline first as this is the key component of the project
# Problem Statement
- This sentence "Furthermore, a byproduct of this project is that the student will gain expertise in cutting-edge ML techniques and learn to use them in the context of high granularity image generation and fast simulation. Moreover, this project can serve as a baseline for future ML pipelines for all experiments at CERN", you can drop as it 
# ML FastSim in Geant4 Training Pipeline
- The title should be changed to ML FastSim Training Pipeline
- model which will later be connected with Inference -> model which will later be used to perform Inference in Geant4.
- The paragraph from "The project's objective is to use Kubeflow to handle the development of a scalable ML ... to Open source, wide community support " should go to the introduction, there you mention the kubeflow pipeline for once and then you can refer to it for the rest of the blog!
- The ML FastSim in Geant4 components -> The ML FastSim components, this small section can be transformed into a single sentence saying that there are 2 workflows/piplines 
- The ML FastSimulation (Training) in Geant4 can be broken down into the following functional components -> this should be a sentence not a title 
- Par04 Project Structure -> this is not the Par04 strucutre (the Par04 is a G4 example where it shows how to use these ML models for inference in C++), so you would rather say ML FastSim project. And this can be converted into a sentence not a title as well as for Refactored Par04 Project into Kubeflow Pipeline (here also remove Par04)
- Please restructure the paragraph Katib YAML Template : a lot of text and long sentences it is hard to follow 
# Kubeflow
- This section can be dropped, you can briefly describe Kubeflow with few sentences in the introduction along with the pipeline
# Kubeflow Pipeline Preparation
- This section can also be dropped and you can instead summarize it into few sentences and use it for the next section
# Pipeline Components of ML FastSim in Geant4 Training -> here drop Geant4 
- Please in this section keep it short and don't go much into details
# Demonstration of Pipeline building steps: this section can also be dropped or summarized to only few short sentences 
# Large Data Handling : here you need to first mention what is the problem and then describe the solution wiht one sentence 
# Katib: the section Understanding Katib YAML can be dropped just mention that a YAML file is used

**Comment by guneetsk99** [comment]
@DalilaSalamani Can you confirm the latest changes

**Review by DalilaSalamani — APPROVED** [review]

**Comment by guneetsk99** [comment]
@agheata 
Can you approve the merge?
Dalila has approved the changes from her side.
Thanks

**Comment by guneetsk99** [comment]
@agheata follow up on the previous comment

File: _gsocblogs/2022/blog_Geant4_GuneetSingh.md
Comment (agheata) [inline _gsocblogs/2022/blog_Geant4_GuneetSingh.md:23]:
```suggestion

# Outcomes from GSOC 2022
```

File: _gsocblogs/2022/blog_Geant4_GuneetSingh.md
Comment (agheata) [inline _gsocblogs/2022/blog_Geant4_GuneetSingh.md:33]:
```suggestion

# Problem Statement
```

File: _gsocblogs/2022/blog_Geant4_GuneetSingh.md
Comment (agheata) [inline _gsocblogs/2022/blog_Geant4_GuneetSingh.md:36]:
```suggestion

# ML FastSim Training Pipeline
```

File: _gsocblogs/2022/blog_Geant4_GuneetSingh.md
Comment (agheata) [inline _gsocblogs/2022/blog_Geant4_GuneetSingh.md:131]:
```suggestion
<img src="https://user-images.githubusercontent.com/43180442/191947451-0f3ade57-4ece-4754-8a87-83623e84e008.png" width="100%">
```

File: _gsocblogs/2022/blog_Geant4_GuneetSingh.md
Comment (agheata) [inline _gsocblogs/2022/blog_Geant4_GuneetSingh.md:335]:
```suggestion
<img src="https://user-images.githubusercontent.com/43180442/191950051-5409641f-8b01-42b5-a06d-f2dc5dfdd5ec.png" width="100%">
```

**Review by agheata — CHANGES_REQUESTED** [review]
Formatting changes

**Comment by agheata** [comment]
@guneetsk99 I'll apply some minor formatting changes myself

**Review by agheata — APPROVED** [review]

**Pull request merged**
